src/pages/Detail/Modal/ModalApply.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const ModalApply = props => {
```

Comment:
```suggestion
const ModalApply = ({closeModal}) => {
```
매개변수 자리에서 바로 구조분해 할 수 있습니다.